### PR TITLE
Fix phpdoc - init layout string argument

### PR DESCRIPTION
**Pull request by PhilippGrashoff**
Currently, passing a string argument to App::initLayout() produces an error:
```
TypeError : array_key_exists() expects parameter 2 to be array, string given
 /var/www/html/atk/ui/vendor/atk4/core/src/Factory.php:57
 /var/www/html/atk/ui/vendor/atk4/core/src/Factory.php:105
 /var/www/html/atk/ui/vendor/atk4/core/src/Factory.php:239
 /var/www/html/atk/ui/vendor/atk4/core/src/DiContainerTrait.php:108
 /var/www/html/atk/ui/vendor/atk4/core/src/DiContainerTrait.php:145
 /var/www/html/atk/ui/src/App.php:432
 /var/www/html/atk/ui/tests/AppTest.php:51

```

This PR will fix it. At the moment, there is only a test as proof that it does not work.

### CODE REVIEW

File: tests/AppTest.php [hunk 1]
```diff
@@ -6,16 +6,19 @@
...
 
 class AppTest extends AtkPhpunit\TestCase
 {
     protected function getApp()
     {
-        return new App([
-            'catch_exceptions' => false,
-            'always_run' => false,
-        ]);
+        return new App(
```
Comment (mvorisek) [inline tests/AppTest.php]: no :)

File: tests/AppTest.php [hunk 2]
```diff
@@ -41,4 +44,12 @@ public function testTemplateClassCustom()
             $app->loadTemplate('html.html')
         );
     }
+
+    public function testInitLayoutWithStringArgument()
+    {
+        $app = $this->getApp();
+        $app->initLayout(Admin::class);
```
Comment (mvorisek) [inline tests/AppTest.php]: maybe we require standard seed, ie. `[Admin::class]`, thus wont'merge if so

File: tests/AppTest.php (hunk 1 above)
Comment (PhilippGrashoff) [reply tests/AppTest.php]: ?

File: tests/AppTest.php (hunk 2 above)
Comment (PhilippGrashoff) [reply tests/AppTest.php]: yes, then this PR would consist of dropping string param. Though, to be honest, a string param seems quite sensible. At least I didn't see code doing more that just passing class or object, e.g. passing extra parameters.

File: tests/AppTest.php (hunk 1 above)
Comment (mvorisek) [reply tests/AppTest.php]: original CS is ok and better - we do not want code to be longer than needed when readability remains

File: tests/AppTest.php (hunk 2 above)
Comment (mvorisek) [reply tests/AppTest.php]: no, we have strict convenction about seeds - always an array with 1st param class name

Comment (PhilippGrashoff) [reply tests/AppTest.php]: That makes life easy, just removed in comment that string is a valid parameter

File: src/App.php [hunk 3]
```diff
@@ -423,7 +423,7 @@ public function terminateJson($output, array $headers = []): void
     /**
      * Initializes layout.
      *
-     * @param string|Layout|array $seed
+     * @param Layout|array $seed
```
Comment (mvorisek) [inline src/App.php:426]: 👍

File: tests/AppTest.php (hunk 1 above)
Comment (mvorisek) [reply tests/AppTest.php]: if you correct this, we can merge the phpdoc change

Comment (PhilippGrashoff) [reply tests/AppTest.php]: Even php CS fixer does not complain about this. But if its so important to you, I will.

File: tests/AppTest.php (hunk 2 above)
Comment (mvorisek) [reply tests/AppTest.php]: Happy now :)

**Review by mvorisek — APPROVED** [review]

**Pull request merged**